react/src/Spinners.js

```diff
@@ -0,0 +1,84 @@
+import RingLoader from 'react-spinners/RingLoader';
+import BarLoader from 'react-spinners/BarLoader';
+import DotLoader from 'react-spinners/DotLoader';
+import ScaleLoader from 'react-spinners/ScaleLoader';
+import ClipLoader from 'react-spinners/ClipLoader';
+import CircleLoader from 'react-spinners/CircleLoader';
+import PuffLoader from 'react-spinners/PuffLoader';
+import GridLoader from 'react-spinners/GridLoader';
+
+export const RingSpinner = ({loading, color, size}) => {
+    const override: CSSProperties = {
+        display: "block",
+        margin: "0 auto"
+    };
+    return <RingLoader color={color} loading={loading} cssOverride={override} size={size} />
+}
+
+export const BarSpinner = ({loading, color, size, style}) => {
```

Comment:
Is there no naming convention that identifies the 'style' argument? This seems different than some of the other styles around it.